projects/igniteui-angular/src/lib/drop-down/drop-down.component.ts

```diff
@@ -147,7 +144,9 @@ export class IgxDropDownComponent extends IgxDropDownBase implements IDropDownBa
     public get selectedItem(): IgxDropDownItemBase {
         const selectedItem = this.selection.first_item(this.id);
         if (selectedItem) {
-            if (selectedItem.selected) {
+            if (this.virtDir) {
+                return selectedItem;
```

Comment:
If **not** virtual and the first item in the selection is **not** `[selected]`, the selection is cleared.

Reply:
Do we have a case where selectedItem != undefined && selectedItem.selected === false && this.virtDir != undefined
Then it will return selected item which is not selcted!